### PR TITLE
shrshinde/isSupported-notifications-check

### PR DESCRIPTION
**Pull request by shrshindeMSFT**
- Adding isSupported check to notifications capability
- Adding unit test cases for notifications capability

### CODE REVIEW

**Review by lakhveerk — DISMISSED** [review]

File: change/@microsoft-teams-js-4b8c81b9-6691-4b9d-bb1a-f8cb66a0c130.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Adding isSupported check and missing unit test cases",
```
Comment (erinha) [inline change/@microsoft-teams-js-4b8c81b9-6691-4b9d-bb1a-f8cb66a0c130.json:3]: Can you please add for which capability this change is adding the unit test cases/isSupported check?

**Review by erinha — APPROVED** [review]
Thanks for adding missing unit tests!

File: packages/teams-js/test/private/notifications.spec.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    it('should not allow calls before initialization', () => {
+      expect(() => notifications.showNotification(showNotificationParameters)).toThrowError(
+        'The library has not yet been initialized',
+      );
+    });
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContexts => allowedContexts === context)) {
+        it(`showNotification should throw error if notifications capability is not supported in runtime config. context: ${context}`, async () => {
+          await utils.initializeWithContext(context);
+          expect.assertions(1);
```
Comment (lakhveerk) [inline packages/teams-js/test/private/notifications.spec.ts:38]: why the expected assertions is just 1 /Are we not counting the assertions in initializeWithContext?

Comment (shrshindeMSFT) [reply packages/teams-js/test/private/notifications.spec.ts:38]: This one is just for catch block to be executed and not for initializeWithContext

**Review by lakhveerk — APPROVED** [review]

**Pull request merged**